src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -116,13 +111,42 @@ private static bool IsNamespaceName(NameSyntax name)
             }
         }
 
+        private void TryClassifyStaticSymbol(
+            ISymbol symbol,
+            Text.TextSpan span,
+            ArrayBuilder<ClassifiedSpan> result)
+        {
+            if (symbol?.IsStatic == true
+                && (!symbol.IsKind(SymbolKind.Field) || symbol.ContainingType?.IsEnumType() == false) // TODO: Since Enum members are always static is it useful to classify them as static?
+                && !symbol.IsKind(SymbolKind.Namespace)) // TODO: Since Namespace are always static is it useful to classify them as static?
+            {
+                result.Add(new ClassifiedSpan(span, ClassificationTypeNames.StaticSymbol));
```

Comment:
so, now does this mean that we can have multiple classifiedSpan for same span? is that okay? have we ever had 2 classifiedSpan with same span?

I think you might need to change ClassifiedSpan to have either properties or attributes and set static as attribute rather than classification type. people will probably argument which should be main classification type and what is attribute, but regardless of that, I think we need the main one and attribute.

I can think of other cases where we would use attributes, like whether a reference is captured or not, whether it is read or write, or boxed and etc.